pkg/framework/host.go

```diff
@@ -565,12 +574,27 @@ func (h *Host) WaitForPodLog(namespace, needle, podName string) error {
 	return microerror.Mask(notFoundError)
 }
 
-func (h *Host) crd(crdName string) func() error {
+func (h *Host) crd(ctx context.Context, name string) func() error {
 	return func() error {
-		// FIXME: use proper clientset call when apiextensions are in place,
-		// `k8sClient.ExtensionsV1beta1().ThirdPartyResources().Get(tprName, metav1.GetOptions{})` finding
-		// the tpr is not enough for being able to create a tpo.
-		return runCmd("kubectl get " + crdName)
+		o := func() error {
+			_, err := h.extClient.ApiextensionsV1beta1().CustomResourceDefinitions().Get(name, metav1.GetOptions{})
+			if apierrors.IsNotFound(err) {
+				return microerror.Mask(err)
+			} else {
+				return backoff.Permanent(microerror.Mask(err))
+			}
+
+			return nil
+		}
+		b := backoff.NewExponential(backoff.ShortMaxWait, backoff.ShortMaxInterval)
+		n := backoff.NewNotifier(h.logger, ctx)
+		err := backoff.RetryNotify(o, b, n)
+
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		return nil
```

Comment:
This is the most important part. It should fix builds like https://circleci.com/workflow-run/f929a4aa-b711-4902-933e-eedde9633670.

Reply:
Very nice!